translations/ui-inventory/en.json

```diff
@@ -23,6 +23,8 @@
   "uniqueName": "Name must be unique",
   "fillIn": "Please fill this in to continue",
   "selectToContinue": "Please select to continue",
+  "showSelectedRecords": "Show selected records",
+  "selectedRecords": "Selected records",
```

Comment:
Let's put them under the instances (e.g. 'instances. showSelectedRecords') namespace near the "instances.rows.select".